.travis.yml

```diff
@@ -27,9 +28,31 @@ install:
   - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then gem install danger danger-junit; fi'
 
 script:
+  - docker-compose exec web bash -c "wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb"
+  - docker-compose exec web bash -c "dpkg -i google-chrome-stable_current_amd64.deb; apt-get -fy install"
+#  - docker-compose exec web bash -c "apt-get install google-chrome-stable chromium-chromedriver"
+  - docker-compose exec web bash -c "whereis google-chrome-stable"
+  - docker-compose exec web bash -c "google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &"
+  - docker-compose exec web bash -c "wget https://chromedriver.storage.googleapis.com/2.35/chromedriver_linux64.zip"
```

Comment:
Should we switch to v73???

https://sites.google.com/a/chromium.org/chromedriver/


https://chromedriver.storage.googleapis.com/index.html?path=73.0.3683.68/

But there's also troubleshooting info on that first page

Reply:
Try `73.0.3683.68` next